portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,164 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+*{
+    box-sizing: border-box;
 }
 
-#greeting-container {
-  margin-top: 20px;
+nav {
+    background-color: black;
+    color: white;
+    text-align: center;
+    font-size: 24px;
+    width: 100%;
+    overflow: hidden;
+    position: sticky;
 }
+
+nav span {
+    float: left;
+    padding: 14px 16px;
+    color: white;
+}
+
+nav a {
+    color: white;
+    text-decoration: none;
+    padding: 14px 16px;
+    float: right;
+
+}
+
+nav a:hover {
+    /* for underline under anchor text */
+    /* text-decoration: underline;
+    color: white; */
+    background-color: green;
+}
+
+.hero {
+    background-image: url('https://lp-cms-production.imgix.net/2019-06/512d769cd0d5e56c238eb975d2325940-japan.jpeg?fit=crop&q=40&sharp=10&vib=20&auto=format&ixlib=react-8.6.4');
+    padding: 5px;
+    margin: 0;
+    height: 100%;
+    top: 0;
+
+    background-attachment: relative;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.hero-text {
+    text-align: center;
+    position: absolute;
```

Comment:
I'm not crazy about using the position rule either. The problem with both floating and position is that they're brittle (meaning minor changes break them easily) and indecipherable (I don't know what this looks like without actually seeing it). I recommend trying flex here as well.